scheduler/utilization_based_host_allocator.go

```diff
@@ -101,6 +101,10 @@ func UtilizationBasedHostAllocator(ctx context.Context, hostAllocatorData HostAl
 		// add up total number of hosts needed for all groups
 		numNewHostsRequired += n
 		numFreeApprox += free
+		if name != "" {
+			taskGroupData.Info.CountFree = free
+			taskGroupData.Info.CountRequired = n
```

Comment:
It doesn't look to me like these values get persisted. They're set only for the current iteration of the loop. They won't make it back up to the host allocator.